### PR TITLE
upload code with auth code 

### PR DESCRIPTION
**Pull request by GurpreetSingh013**
upload code with auth code

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            }
+            
+        }
+
+    } 
+})
+
+const maxSize = 10* 1000 * 1000; //10mb size
+    
+var upload = multer({
```
Comment (officialanurag) [inline app.js:47]: use const instead

File: app.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    storage: storage, 
+    limits: { fileSize: maxSize }, 
+    fileFilter: function (req, file, cb){ 
+        var filetypes = /mp4|mp3/; 
+        var mimetype = filetypes.test(file.mimetype); 
+        var extname = filetypes.test(path.extname(file.originalname).toLowerCase()); 
+
+        console.log(file.mimetype);
+        console.log(path.extname(file.originalname).toLowerCase());
+
```
Comment (officialanurag) [inline app.js:57]: use const if not re-assigned

File: app.js (hunk 1 above)
Comment (GurpreetSingh013) [reply app.js]: done

File: app.js (hunk 2 above)
Comment (GurpreetSingh013) [reply app.js]: done

**Pull request merged**